pkg/controller/secretannotator/azure/reconciler.go

```diff
@@ -77,7 +77,7 @@ func (r *ReconcileCloudCredSecret) Reconcile(request reconcile.Request) (reconci
 	}
 
 	r.Logger.Info("Platform is azure: using passthrough")
-	err = r.updateSecretAnnotations(secret, constants.PassthroughAnnotation)
+	err = r.updateSecretAnnotations(secret, constants.MintAnnotation)
```

Comment:
We should have a big TODO here, we need to implement dynamic checks to verify if we have sufficient perms to create new apps/SPs, if not do we have enough to passthrough, and if not error out.

Any word from Nick if it's ok to ship with this mint assumption?

Reply:
+100 on dynamic checking.